client/src/components/layout/HelForm.jsx

```diff
@@ -102,13 +102,15 @@ const HelForm = ({ submitForm, submitted, user }) => {
   };
 
   const handleCourseAddition = e => {
-    if (!humanitiesCourses.includes(e.target.innerHTML)) {
+    let event = (e.target.innerHTML).toLowerCase().split(" ");
+    event = event[0]+" "+event[1];
+    if (!humanitiesCourses.includes()) {
       if (isEditingRowAtIndex !== -1) {
-        humanitiesCourses[isEditingRowAtIndex] = e.target.innerHTML;
+        humanitiesCourses[isEditingRowAtIndex] = event;
         setFormData({ ...formData });
       } else {
         addCourseToList();
-        humanitiesCourses[isEditingRowAtIndex] = e.target.innerHTML;
+        humanitiesCourses[isEditingRowAtIndex] = event;
```

Comment:
This change is making the shorter code only get displayed on the form. We don't want that. It needs to be displayed properly for the user, only shortened while saving. Maybe do this in the api endpoint?

Reply:
fair, will do. I'll need to clone event, and wasn't paying attention

Reply:
Done, please check the latest